### DISCUSSION TITLE
2008/02/19/%E5%9B%9B%E5%A4%A7%E5%90%8D%E8%91%97%E7%9A%84%E8%AF%84%E6%B3%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#209] [Announcements]
# 2008/02/19/%E5%9B%9B%E5%A4%A7%E5%90%8D%E8%91%97%E7%9A%84%E8%AF%84%E6%B3%A8

我找同事帮忙，下载了全套的百家讲坛，于是没事的时候就听听。文学名著的评析是百家讲坛中比重较大的一类主题。而在这类主题中，中国的古典四大名著更是重中之重。尽管上百家讲坛的都是有点研究成就的人，但从我的角度来看，他们的水平仍然良莠不齐。

https://blog.qizhen.xyz/2008/02/19/%E5%9B%9B%E5%A4%A7%E5%90%8D%E8%91%97%E7%9A%84%E8%AF%84%E6%B3%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

妍
2009年02月6日 19:06 编辑
倒是推荐《悟空传》，那是真真得好看

回复
Qizhen
2009年02月4日 09:41 编辑
唐僧为什么要取经 小时候看《西游记》，感觉妖怪是最笨的，哪里需要孙猴子去打，自己就笨死了，明明唐僧落到手中，却总是不忙吃，结果总是被神仙们连窝铲除，连山贼这个很有前途的职业都做不成。要我是妖怪，先把唐僧吃了，吃一口就长生不老，全吃了还不功力暴增，金刚不坏呀，那时候再去跟孙猴子和各路神仙PK，其奈我何？　　　　经历的事情多了，才知道问题不那么简单。《西游记》虽然讲的是神仙妖魔世界，本质上仍然是一个政治故事。既然是政治，决定能否吃掉对方的，自然不是嘴，而是权力。从政治角度来说，唐僧根本就不是一种适于食用的生物，为什么呢？因为在政治生态圈中，妖怪们看似强大，却处于食物链的低端，而唐僧反之，看似文弱，却处于食物链的高端。“吃唐僧一口肉可长生不老“，原本就是一句政治谣言。这样说，大家会越看越乱，还是让我们从孙猴子大闹天宫说起。　　　　孙悟空的造反，根本上说是源于天庭人才选拔、晋升与使用制度的缺位，并非象某些站在老板角度写的书所言，什么猴子心火太旺，自视太高，不懂适应社会等等。如果说猴子有什么不懂的地方，就是他不懂政治，没有学习过几个代表，不懂得体现自身的先进性，故而在生存斗争中无战不胜，而在政治斗争中则始终无法证明自己执政的合法性。　　　　孙猴子出国求学归来，虽然作为一方黑社会老大，以强索金箍棒，强销死籍等等行动，树立了经典恐怖主义权威形象，但毕竟是在政治生态链低端竞争而已。在强销死籍，彻底解决生存等问题之后，根据马斯洛的需求层次论，他需要自我成就，但他这种仙，只配叫“妖仙 “，天庭当局是不承认的，他的自我成就感是无法得到满足的。象猴子这样的妖仙或妖怪，《西游记》中比比皆是，散落在这个星球的各个角落，他们都面临着孙猴子一样的自我实现困境。天庭既然拒不提拔这种草根阶层人才，可以想象，只要妖仙们想自我实现，就只能造反，或者招安，没有第三路可走，因此，没有孙悟空造反，也会有别的妖仙起事，这跟《水浒》中好汉的下场没有区别。值得一提的是六耳弥猴，它看到了孙悟空当年激进手段的失败，于是启用最高级的病毒程序，成功地把自己复制为孙悟空，要死要活地混入西天取经的革命队伍，企图采用和平演变手法，达到晋升的目的。孙悟空或许会奇怪，取经有什么好，干嘛那么辛苦来伪装自己。说白了，草根阶层只要有能力有愿望，谁不想出人头地，由被统治者成为统治者呢？　　　　对于猴子这样的草根豪杰，玉帝第一反应是“那路神将下界收伏？“，也就是说，在玉帝的脑海里，根本没有危机感，他并不认为需要扩大统治基础，当然更谈不上对人才进行什么识别和选拔。反之，太白金星是个人物，他建议道：“臣启陛下，可念生化之慈恩，降一道招安圣旨，把他宣来上届，授他一个大小官职，与他籍名在箓，拘束此间，若受天命，后再升赏；若违天命，就此擒拿。一则不动众劳师，二则收仙有道也。“这个建议极具深远的政治意义，一是革新人才体制，化解民间积怨，二是扩大统治阶级基础，问题在于玉帝是个见子打子的主儿，根本不懂得改革要配套进行，其后对孙悟空的使用不当，导致了大闹天宫。更导致了如来不得不另起炉灶，通过“唐僧取经“重建政治生态圈的大谋略。　　　　《西游记》第四回这样描写猴子对天庭来使的反应：金星领了旨，出南天门外，按下祥云，直至花果山水帘洞。对众小猴道： “我乃天差天使，有圣旨在此，请你大王上届，快快报知！“洞外小猴，一层层传至洞天深处，道：“大王，外面有一老人，背着一角文书，言是上天差来的天使，有圣旨请你也。“美猴王听得大喜，道：“我这两日，正思量要上天走走，却就有天使来请。“叫：“快请进来！“猴王急整衣冠，门外迎接。由此可见，孙悟空哪里想造反，只不过是想出人头地自我实现耳。如果玉帝能多动动脑袋，对猴子使用得当，就不会有那么多破事发生了。但玉帝没念过MBA，实在不懂管理，对猴子的使用和管理上步步失当。猴子也没地方跳槽，自然要造反了。　　　　第一次人事任命，由于天庭人浮于事，各宫各殿，各方各处，都不少官，只是御马监缺个正堂管事，玉帝随口就让猴子做个“弼马温“，在人事安排上何等随意，完全不考虑适才适用原则，也谈不上什么对人才的重视与重用，难怪猴子知道真相后大为不满。孙猴子在这活计上干得不懒，宇宙一流水准，从这点说，他是个好员工，却活在一个不好的体制中。　　　　第二次人事任命，猴子反下天庭，自立为“齐天大圣“，齐天者，意思就是并不造反，只是要求自我成就感而已。玉帝闻言，惊讶道：“这妖猴何敢这般狂妄！着众将即刻诛之。 “太白金星又建议加他个空衔，名是齐天大圣，只不与他事管，不与他俸禄，且养在天壤之间，收他的邪心，使不生狂妄，庶乾坤安靖，海宇得清宁也。这一次，太白金星出的其实是馊主意了，如果每个妖仙或妖怪都学孙猴子这样要官，而且都能如愿以偿，那爵位还有什么尊荣可言？天庭整个体制又何以正常运转？玉帝大概是苦修几亿年把脑子锈掉了，居然“依卿所奏。“到底玉帝是凭什么在进行管理，真是天才知道。正因为玉帝在孙悟空的挑战面前轻易让步，让猴子觉得一闹就有甜果果吃，才会有后来大闹天宫，直接要求做皇帝。　　　　第三次人事任命就更奇怪了。多管闲事的许旌阳真人启奏道：“ 今有齐天大圣，无事闲游，结交天上众星宿，不论高低，俱称朋友。恐后闲中生事，不若与他一件事管，庶免别生事端。“玉帝闻言，即时宣诏。那猴王欣欣然而至，道：“陛下，诏老孙有何升赏？“玉帝道：“朕见你身闲无事，与你件执事。你且权管那蟠桃园，早晚好生在意。“大圣欢喜谢恩，朝上唱喏而退。他等不得穷忙，即入蟠桃园内查勘。大家应该明白，这完全是玉帝无事找事，自找麻烦。把猴子收上天的本意，就是让他不闹事，目的已经达到了，怎么又想起让他做事呢？做别的也就罢了，偏偏让他去管蟠桃园，猴子爱吃桃，三岁小孩都知道，玉帝居然不知道。难道玉帝是有意锻炼猴子的职业操守，以备大用？看不出来，反而有种陷人以罪，免得诛之无名的味道。有些人说孙悟空不是好员工，理由就有他监守自盗偷吃蟠桃，我不这样认为，玉帝的这个任命，是违反“猴性“的，就象一些企业的规度违反人性一样，不能怪员工不遵守，只能说老板脑子坏掉了。　　　　其后发生的一连串事件，起因皆在王母蟠桃嘉会没有请齐天大圣，严重损伤猴子自尊心。直到此时，猴子仍没有想到造反，他在丹满酒醒之后道：“不好！不好！这场祸，比天还大；若惊动玉帝，性命难存。走！走！走！不如下界为王去也！“所以说，孙悟空的大闹天宫夺权行动，纯粹是由于生存受到威胁，一不做二不休，把生存斗争提高到政治斗争，可惜猴子道行不够，鬼王将军也不是梁山军师，无法证明自己夺权的合法性，因而既无法建立广泛群众基础，也无法得到如来的认同，造反失败也就不可避免了。　　　　玉帝能当上天庭最高行政长官，用如来的话讲：“他自幼修持，苦历过一千七百五十劫。每劫该十二万九千六百年。你算，他该多少年数，方能享受此无极大道？“似乎是说，玉帝没有别的本事，就是挂机在线修炼时间足够长。不懂政治的猴子眼里，玉帝不过是多年媳妇熬成婆，自然不服气，但从《西游记》中潜伏的小线索来看，玉帝应该是在无限长期的政治修炼中，逐渐成为一个各方各派都可以接受的荣誉长官。且看天蓬元帅醉酒戏嫦娥被打了二千锤再贬下凡尘，卷帘大将失手打碎了玻璃盏，被打八百，贬下界来，又教七日一次，将飞剑来穿胸胁百余下，处罚何等严酷，而奎木狼与披香殿玉女私通，玉帝只收了他金牌，贬他去兜率宫与太上老君烧火，带俸差操，有功复职，无功重加其罪，处罚又何之轻。说白了，天蓬元帅也好，卷帘大将也好，都没有什么靠山，不如奎木狼仍二十八星宿之一，有团队做后盾。玉帝对猴子、八戒、沙僧这种没有后山的动物，是从不来不会手软的，而对有后台的动物，哪怕是个神仙跨下坐骑，也从来高举轻打。　　　　天庭政治结构，貌似二元对等，行政事务管理由玉帝负责，另有一个平行的宗教事务管理层，由如来总负责，表面上互不相关，其实不然，如来时刻在关注和指导行政事务，当然都是大方向大原则的问题，比如平息孙猴子dong乱这样的大事，危急时刻还得如来这样的老同志亲自出手。因此，可以认为，如来的政治地位比玉帝高一个等级，天上地下他最大。也就是说，生物链的顶端，是如来，他可以决定一切。　　　　对于玉帝治理下的种种弊端，如来并非不知，只要不威胁到政权，只是不管罢了，但事情竟然发展到孙悟空造反，直接危及政权稳定，如来不能不对整个治理结构重新思考。经过深思，他下定决心，必须选拔新一代领导人，并重新梳理各方力量，该灭的灭，该升的升，扩大统治基础，使之能代表更多的阶层，把政权危机消弥于无形。　　　　君不密失其国，臣不密失其身，如来当然懂这个道理。因此，经过考查，他选定了座下弟子金蝉子作为新的接班人培养，借口他不听讲座，贬下凡去，让他投胎成为未来的唐僧。然后在盂兰盆会上借讲经的机会，言道有三藏真经，可以劝人为善，需得东土寻一个善信．教他苦历千山，远经万水，到如来处求取真经，永传东土，劝他众生。这个所谓东土善信，当然就是金蝉子。当时观音菩萨就自告奋勇道：“弟子不才，愿上东土寻一个取经人来也。“ 如来见了，心中大喜，道：“别个是也去不得，须是观音尊者、神通广大，方可去得。“菩萨道；“弟子此去东土，有甚言语吩咐？“如来道；“这一去。要踏看路道，不许在霄汉中行，须是要半云半雾；目过山水，谨记程途远近之数，叮咛那取经人。“　　　　为什么观音自告奋勇呢？因为她也是这个接班人计划的制定者和实施者之一，亲自送金蝉子投胎的人是谁？观音大士也。她去东土寻那个取经人，自然万无一失。为什么如来心中大喜，还特意嘱咐观音此去要踏看道路呢？因为观音神通广大，足智多谋，深通佛意，足以完成选拔新一代领导人和重新梳理各方力量这两个目标。也就是说，唐僧的取经也好，孙悟空等三人做徒弟也好，都是如来和观音一手安排，同时，八十一难也早就安排妥当。　　　　唐僧，即金蝉子，作为新一代接班人，有什么条件呢？换句话说，孙悟空为什么就不能成为接班人呢？说白了就是出身问题，孙悟空不过一个猴子，无根无基，修行不过三百来年，而唐僧乃金蝉子投胎，此前已经十世修行，转世之后，自幼即在寺庙修行，此所谓根正苗红，信念坚定，年资充足，合法性强也。据称唐僧是什么完美型领导者，悟空是力量型员工，什么团队组合等等，那是接近胡说的戏说，领导说你行，你就行，不行也行，领导说你不行，你就不行，行也不行。唐僧就是这样，在猴子的dong乱平息后，为了政权的长治久安而被扶上位的。　　　　唐僧作为天定的取经团队头儿，从第一次被妖怪捉去“几乎唬死“，到最后一次被捉去仍然只会战战兢兢大喊“大王饶命“，不知是该佩服他，还是该BS他。按常理，一个人历经多少年的磨难，意志应该磨炼得坚强无比，就算天生是个软蛋，但只要没有得失忆症，他总该记得自己一路上反反复复地被妖怪捉去，又反反复复地被神仙解救，每次都有惊无险，也早该有种无所谓的态度了吧。每次看到唐僧被妖怪捉去，我就不由自主地想起吴孟达先生的名句： “又升天了！咦？为什么要说又呢？“唐僧先生一次又一次又一次被妖怪拿住又一次又一次又一次得救之后，面对下一个妖怪伸过来的魔爪，他应该是一把抓住紧紧地握手，好象地下党员见到组织，应该很热烈地说：“又来了！咦，我为什么要说又呢？麻烦你烧点水，我洗个澡先！“，他蛮可以谈笑风生地与妖怪们商量唐僧有几种吃法，反正性命铁定无忧，何必每次都要装出窝囊废的模样呢？我们只能说，唐僧明明知道在政治生物链上，妖怪们跟自己完全不是一个等级，但他仍然兢兢业业地扮演弱者，只是为了把戏演得真一点，避免玉帝这样的政治对手的警觉，也博取更大的同情。　　　　唐僧是领导者么，是一个优秀的领导者么？我没看出来。整部书里只有一个领导者，总设计师，总导演，就是如来，在他设定的旅游线路上，设定的情节里，傻子坐在唐僧那个位子，也能把真经取到手。至于唐僧那三个徒弟，如来的使用手法，让人不禁想起唐太宗临死前对徐世碛的运用。据说太宗皇帝为了让儿子高宗有可用之臣，特地找个岔子把徐世碛关进死牢，观察他的表现，考验合格后等高宗继位再放出来重用，这样一来，徐世碛不但不敢对新皇帝摆老臣的谱儿，反而会无尽感激新皇帝的救命之恩，岂敢不竭忠尽智死而后已。八戒沙僧都是玉帝下旨干掉的，悟空则是如来亲自下手，再由唐僧把他们救出来，手法如出一辙，所谓帝王心术，此其一耳。　　　　话说观音与木吒到了长安，长安城各庙神抵都来参见，菩萨道：“汝等不可走漏消息。我奉佛旨，特来此处寻访取经人。借你庙宇，权住几日，待访着真僧即回。“众神各归本处，把个土地赶到城隍庙里暂住。看来观音虽然慈悲，在接待规格上也是一点都不含糊的。顺便一说。　　　　唐僧作为未来领导者，不足之处，一是没有神际关系网，二是没有政绩，三是没有班底，四是没有威望。要解决这些问题，十万八千里的取经，足矣。唐僧的嫡系班底就是三个徒弟，全部是体制外动物，而且全部是政治链低端动物，这是如来有意安排的，说穿了，三个徒弟代表的是广大中下级神仙妖怪，简称三个代表，这就给了广大体制外人物一个晋升的希望，一个晋升的路子。　　　　取经路上，为什么那些妖怪许多是天上的座骑、童子下凡变的，而孙猴子为什么动不动就上天去搬救兵，说白了，联络各部关系，让各位大仙认识一下唐僧，在战火中培养出革命的感情，以使唐僧接触各方神圣，建立神际关系网，以便将来登基顺利。事实也表明，取经的一路上，所有的重量级政治人物都出面了，象什么四圣试禅心，黎山老母、南海菩、普贤、文殊全部出场，用意还不明显么？　　　　至于政绩，是历经苦难取得真经么？非也非也，取经这种事，在神仙眼里简直就是小儿科，噱头而已，真正的政绩是唐僧们一路灭了无数妖魔，又收伏了无数妖魔，彻底解决了政权不稳定因素，又为天庭提拔了一批在野力量，还为唐僧建立了一个广泛的群众基础。想想一个孙悟空当年就闹得天上鸡犬不宁，只有请出如来才彻底解决，而唐僧们一路杀妖伏魔，这个水平，这个政绩，这个威望，啧啧，不言自明。　　　　为什么说“吃了唐僧肉长生不老“是一句谣言呢？第一个说这话的妖怪是谁，白骨精，他在云端里，踏着阴风，看见长老坐在地下，就不胜欢喜道：“造化！造化！几年家人都讲东土的唐和尚取大乘，他本是金蝉子化身，十世修行的原体。有人吃他一块肉，长寿长生。真个今日到了。“ 后来的金角大王也教育弟弟说：“你不晓得。我当年出天界，尝闻得人言：唐僧乃金蝉长老临凡，十世修行的好人，一点元阳未泄，有人吃他肉，延寿长生哩。“观音不是在长安嘱咐众神“汝等不可走漏消息“么？怎么现在变成地球人都知道了？是谁传出这个信息呢？还不是观音自己。她故意传出这个消息，引诱各方妖魔聚集到西天取经路上，这叫引蛇出洞一网打尽，省得满世界去剿去抚搞得麻烦。　　　　为什么我敢说唐僧取经不过是一个小把戏，一个噱头而已，为什么我敢说唐僧之取经，不过是为了政权稳定，为了接班人的培养计划呢？所有的谜底都在《西游记》的结尾揭开。依如来的说法，西天取经的原意，是为了普渡众生，而唐僧历经艰苦取得真经，就应该留在大唐，象历史上的玄奘大师那样，翻译著述传播，为普度众生奉献自己的一生才对。但小说告诉我们，金刚把唐僧们送到长安，停在空中叫道：“圣僧，此间乃长安城了。我们不好下去，这里人伶俐，恐泄漏吾像。孙大圣三位也不消去，汝自去传了经与汝主，即便回来。我在霄汉中等你，与你一同缴旨。“看看，传了经给唐王就回西天就职去，一刻都不能耽误，哪来普度众生的样子？书中写道唐王请三藏长老讲几卷经，长老捧卷登台，方欲讽诵，忽闻得香风缭绕，半空中有八大金刚现身高叫道：“诵经的，放下经卷，跟我回西去也。“这底下行者三人，连白马平地而起，长老亦将经卷丢下，也从台上起于九霄，相随腾空而去。于是普度不过是丢下经就走，取经人各归神位，形成新的政治派别，为将来的新老交接定下基调。　　　　唐僧师徒回到灵山之后，如来分封四人，道：“圣僧，汝前世原是我之二徒，名唤金蝉子。因为汝不听说法，轻慢我之大教，故贬汝之真灵，转生东土。今喜皈依，秉我迦持，又乘吾教，取去真经，甚有功果，加升大职正果，汝为旃檀功德佛。孙悟空，汝因大闹天宫，吾以甚深法力，压在五行山下，幸天灾满足，归于释教，且喜汝隐恶扬善，在途中炼魔降怪有功，全终全始，加升大职正果，汝为斗战胜佛。猪悟能，汝本天河水神，天蓬元帅，为汝蟠桃会上酗酒戏了仙娥，贬汝下界投胎，身如畜类，幸汝记爱人身，在福陵山云栈洞造孽，喜归大教，入吾沙门，保圣僧在路，却又有顽心，色情未泯，因汝挑担有功，加升汝职正果，做净坛使者。“八戒口中嚷道： “他们都成佛，如何把我做个净坛使者？“如来道：“因汝口壮身慵，食肠寛大。盖天下四大部洲，瞻仰吾教者甚多，凡诸佛事，教汝净坛，乃是个有受用的品级，如何不好！沙悟净，汝本是卷帘大将，先因蟠桃会上打碎玻璃盏，贬汝下界，汝落于流沙河，伤生吃人造孽，幸皈吾教，诚敬迦持、保护圣僧，登山牵马有功，加升大职正果，为金身罗汉。“又叫那白马：“汝本是西洋大海广晋龙王之子，因汝违逆父命，犯了不孝之罪，幸得皈身皈法，皈我沙门，每日家亏你驮负圣僧来西，又亏你驮负圣经去东，亦有功者，加升汝职正果，为八部天龙马。“长老四众，倶各叩头谢恩。　　　　如来对四人的考评、授职和使用，有根有据，合情合理，量才适用，较之玉帝一会儿弼马温，一会儿齐天大圣，一会儿管蟠桃园，水平高下不言自明。纵观整个西游故事，如来作为最高领导，计算之精，安排之密，成果之丰硕，足令人高山仰止，回味无穷。　　　　附记：本文纯粹戏说，也可当作胡说。当年江主席时代，听过我这种解释的人，都说在讽刺现实，我只能说巧合而已。当然虽是巧合，倒也说明神仙与凡界，古代与当今，在选拔与培养领导人上，模式也差不多。仅供各位朋友一乐。另外一点，据说有两本书，争论孙悟空是个什么样的员工的问题。我没读过，也没兴趣读，仅从书名上说，纯粹就是胡扯，员工怎么分好坏？按什么标准？问题是在领导者身上，使用不当，再“好“的员工也会变“坏“，反之亦然。

回复
秋
2008年03月6日 09:43 编辑
我好像一本也没看过,就只是看的电视~而且还看得不全~
嘻嘻嘻~~~~~~~~~~~~~

回复
acca
2008年02月19日 14:05 编辑
唐僧取景是为了唐王江山
而西游记一方面是为了讽刺官场,另一方面也是宣扬了不屈不挠的斗争精神.
 
孙悟空 猪八戒 沙僧 唐三藏分别指代了4种不同的人物性格…
 

回复
arrowpig
2008年02月19日 12:29 编辑
QQ，不要怕过年，你猪年可不是一般的兴啊！你推荐的文章我一定要看的，上次推荐的《动物职场守则》我看了终于知道自己不属羊，而是属驴啊～～～但是世界上最无奈的事莫过于明明知道自己是驴，却也无力改变。算了，驴就驴了～～

回复
